src/templates/admin/elements/fundref/fundref.html

```diff
@@ -149,15 +150,37 @@
 
                                     const funder_grant_textbox = document.createElement('input');
                                     funder_grant_textbox.setAttribute('type', 'text');
-                                    funder_grant_textbox.id = 'grant_number';
-                                    funder_grant_textbox.name = 'grant_number';
+                                    funder_grant_textbox.id = 'funding_id';
+                                    funder_grant_textbox.name = 'funding_id';
                                     funder_grant_text.appendChild(funder_grant_textbox);
 
                                     const funder_grant_label = document.createElement('p');
                                     funder_grant_label.classList.add('help-text');
                                     funder_grant_label.innerText = '{% trans funder_form.fields.funding_id.help_text %}';
                                     funder_grant_text.appendChild(funder_grant_label);
 
+                                    // Add the funder statement ID
+                                    const funder_statement_modal_row = document.createElement('div');
+                                    funder_statement_modal_row.classList.add('row');
+                                    add_funder_form.appendChild(funder_statement_modal_row);
+
+                                    const funder_statement_text = document.createElement('div');
+                                    funder_statement_text.classList.add('large-12');
+                                    funder_statement_text.classList.add('columns');
+                                    funder_statement_modal_row.appendChild(funder_statement_text);
+
+                                    const funder_statement_textarea = document.createElement('textarea');
+                                    funder_statement_textarea.id = 'funding_statement';
+                                    funder_statement_textarea.name = 'funding_statement';
+                                    funder_statement_text.appendChild(funder_statement_textarea);
+
+                                    const funder_statement_label = document.createElement('p');
+                                    funder_statement_label.classList.add('help-text');
+                                    funder_statement_label.innerText = '{% trans funder_form.fields.funding_statement.help_text %}';
+                                    funder_statement_text.appendChild(funder_statement_label);
```

Comment:
These form inputs all [need labels in order to be accessible](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names?utm_source=devtools&utm_medium=a11y-panel-checks-text-label#form_elements_must_be_labeled). I know you're just adding one textarea and the other inputs are pre-existing, but you're changing quite a few things about how funders are edited, so it seems like the right time to fix this form.

Idea: Use the `foundation` template filter to generate empty HTML on this modal from an empty instance of the new Django form, and then use JS to populate the values in the bit above? Might also have the bonus of cutting down on this verbose JS code.